### DISCUSSION TITLE
Connection problems VR700

### DISCUSSION THREAD

**Discussion by Idskov** [BeamCtrl/Airiana#20] [General]
Hi.

Awesome project, just what I was looking for to get my Systemair VR-700DK to be a bit smarter and work with my HomeAssistant.
But unfortunately I am unable to get communication up and running.

I  have the older model main-board (PCU-EC v2.40 27.10.2008) that does not have the RJ45 in the middle, but instead has A B GND 12v in a screw-terminal
![image](https://user-images.githubusercontent.com/64523817/132298723-35761f93-e622-4ffd-978d-c23500bd766c.png)

Do you have any experience in running Airiana with this board or do I need to upgrade to the newer PCU-EC4 board first?
![image](https://user-images.githubusercontent.com/64523817/132298971-f53e9b13-b52c-4297-ae0f-5064588d2c19.png)

**Comment by BeamCtrl** [comment]
Glad to see you take an interrest :D. I don’t see why there would be an issue Using terminal connectors instead of a RJ45. instead it makes finding a cable easier. Have you read the previous disc. On connection issues? What have you tried so far? 
/Daniel
